.github/workflows/add-release-reviewers.yml

```diff
@@ -0,0 +1,28 @@
+name: Add reviewers to release PRs
+
+on:
+  pull_request_target:
+    branches: ['master', 'next']
+    types: ['labeled']
+
+permissions: {}
+
+jobs:
+  add-reviewers:
+    if: ${{ github.event.label.name == 'release' }}
+    # Tests that label is added on the PR
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
```

Comment:
This is probably not going to work. 🙈 
Have you checked https://github.com/cli/cli/issues/7671? 🤔

Reply:
interesting ... need to look into this ... thanks for pointing it out

Reply:
This seems promissing
https://github.com/cli/cli/issues/4844#issuecomment-1825948781

Reply:
I used the curl approach for the API, since the gh cli seemingly does not allow for passing in an array of strings 🤷🏼

Reply:
Well, it might still fail as the suggestion here: https://github.com/cli/cli/issues/4844#issuecomment-1825948781 specifies the write permissions. 🤔

Reply:
yeah ... it did fail, but for another reason! :D